src/api/app/components/request_decision_component.html.haml

```diff
@@ -0,0 +1,25 @@
+= form_tag({ action: 'changerequest' }, remote: true, id: 'request_handle_form') do
+  = hidden_field_tag(:number, @bs_request.number)
+  = text_area_tag(:reason, nil, placeholder: 'Please explain your decision...', rows: 4, class: 'w-100 form-control mb-2')
```

Comment:
I'd like to try something UX wise, only show this text box in a modal for declining requests.

Reply:
```
[2] pry(main)> HistoryElement::RequestAccepted.where(comment: '').count
=> 122559
[3] pry(main)> HistoryElement::RequestAccepted.where.not(comment: '').count
=> 622526
```

Reply:
```
[6] pry(main)> HistoryElement::RequestDeclined.where(comment: '').count
=> 5639
[7] pry(main)> HistoryElement::RequestDeclined.where.not(comment: '').count
=> 347013
```

Reply:
We could also enforce having a reason for declining a request. What do you think? Modals on mobile are not so great.

Reply:
I wouldn't enforce this as you can say this also in other ways like comments...

Reply:
I will create something in the backlog for this experiment :)

Reply:
https://trello.com/c/tThO3QNV/2684-experiment-only-allow-commenting-decision-when-declining-a-request